### DISCUSSION TITLE
Not using znap-repos

### DISCUSSION THREAD

**Discussion by YenLegion** [marlonrichert/zsh-snap#143] [Q&A] [answered]
I moved to a new machine and copied over my znap-repos file into ~/.config/zsh (where I keep my zsh plugins). When I used it in the past I could type znap status && znap pull and I was all setup with the plugins from my old machine, but it is currently not working. Does znap still use this file? Thanks.

**Comment by marlonrichert** [comment, marked as answer]
I removed that feature in 650c2dce0c5a201d7fa3413e74d8353b8c4bcd68. I was never completely satisfied with the way it worked and it became superfluous, now that all `znap` commands dealing with repos automatically call `znap clone`.
